config.py

```diff
@@ -42,6 +42,15 @@ class Config:
 
     VCAP_SERVICES = None
 
+    DM_FRAMEWORK_TO_ES_INDEX_MAPPING = {
+        'g-cloud-9': {
+            'services': 'g-cloud-9'
+        },
+        'digital-outcomes-and-specialists-2': {
```

Comment:
What about our DOS1 briefs which are able to change status still? Also, I assume both DOS1 and DOS2 briefs are being indexed together in the same index rather than in two different indexes or will the FE be searching through two indexes combined?

Reply:
So actually yes, dos1 and dos2 briefs will be going into the same index as they are both compatible with the same mapping.

Reply:
- [ ] (And we should therefore add DOS1 into this mapping?)